src/vm/methodtablebuilder.cpp

```diff
@@ -7667,6 +7704,23 @@ MethodTableBuilder::PlaceInterfaceMethods()
                 PlaceMethodFromParentEquivalentInterfaceIntoInterfaceSlot(itfSlotIt, pCurItfEntry, &rgInterfaceDispatchMapTypeIDs, dwCurInterface);
             }
 #endif
+
+            /*
```

Comment:
Please don't check in commented out code without a comment as to what's going on here.